pkg/compose/build_buildkit.go

```diff
@@ -45,7 +45,7 @@ func (s *composeService) doBuildBuildkit(ctx context.Context, service string, op
 		response, err = build.Build(ctx, nodes,
 			map[string]build.Options{service: opts},
 			dockerutil.NewClient(s.dockerCli),
-			confutil.ConfigDir(s.dockerCli),
+			confutil.NewConfig(s.dockerCli),
```

Comment:
cc @crazy-max I think this is the right replacement, but just in case other changes are needed

Reply:
Yes that's right!